extension/intents/forms/formsContentScript.js

```diff
@@ -89,11 +89,7 @@ this.dictationContentScript = (function() {
     setTimeout(() => {
       if (el.hasAttribute("contenteditable")) {
         // eslint-disable-next-line no-unsanitized/property
```

Comment:
This comment can be removed now, since it doesn't use innerHTML

Reply:
Done